tests/pkg/fixtures/formation_queries.go

```diff
@@ -85,8 +85,8 @@ func UnassignFormationWithTenantObjectType(t require.TestingT, ctx context.Conte
 	return &formation
 }
 
-func CleanupFormationWithTenantObjectType(t require.TestingT, ctx context.Context, gqlClient *gcli.Client, in graphql.FormationInput, tenantID, parent string) *graphql.Formation {
-	unassignRequest := FixUnassignFormationRequest(tenantID, string(graphql.FormationObjectTypeTenant), in.Name)
+func CleanupFormationWithTenantObjectType(t require.TestingT, ctx context.Context, gqlClient *gcli.Client, name string, tenantID, parent string) *graphql.Formation {
```

Comment:
I presume it was because from the `FormationInput` we only use the name anyway, so constructing an `FormationInput` each time we don't have one isn't very convenient and easily readable.